backend/tests/integration-async/migrator/test_migrator_tasks.py

```diff
@@ -596,6 +598,32 @@ def assert_migrator_task():
     retry_until_pass(assert_migrator_task)
 
 
+def test_migrate_recids_from_mirror_external_push_enabled(
```

Comment:
@oguzdemirbasci if you move these tests on `integration` and mock the `run_hal_push` you will be able to `assert_called_once`. This test unfortunately doesn't test much :/